lib/install.js

```diff
@@ -272,8 +272,10 @@ Installer.prototype.run = function (cb) {
         [this, this.saveToDependencies])
     }
   }
-  postInstallSteps.push(
-    [this, this.printInstalled])
+  if (npm.config.get('loglevel') !== 'silent') {
```

Comment:
[ls.js#L95](https://github.com/npm/npm/blob/b2a46693add048fc1468905b6afa254d75702308/lib/ls.js#L95) seems to be the offending line.

@iarna At first I thought those wrote to a file, but seeing that they are logged with `console.log()`, I think the issue is that they are not properly using the `npmlog` utility.